assets/js/app.js

```diff
@@ -13,6 +13,7 @@ import "../css/app.scss"
 //     import socket from "./socket"
 //
 // CoreJS
+import "../vendor/js/bootstrap.bundle.min.js"
```

Comment:
Does this require jquery? 🤔

Reply:
@rosle Bootstrap 5 does not require jquery